pkg/scheduler/api/shared_device_pool.go

```diff
@@ -73,8 +75,32 @@ type Devices interface {
 // make sure GPUDevices implements Devices interface
 var _ Devices = new(gpushare.GPUDevices)
 
-var IgnoredDevicesList []string
-
 var RegisteredDevices = []string{
 	GPUSharingDevice, vgpu.DeviceName,
 }
+
+var IgnoredDevicesList = ignoredDevicesList{}
```

Comment:
`IgnoredDevicesList` is a global var and seems no place will remove items from it, is this correct?

Reply:
Sorry, I didn't fully understand your question.

The only place where the IgnoredDevicesList is currently being overwritten (cleared and filled with new values, currently always constant) is [here](https://github.com/volcano-sh/volcano/blob/master/pkg/scheduler/api/node_info.go#L348).

If you mean that using a global variable is not justified, then I may agree with you.

Reply:
Sorry, I didn't notice your `Set` method has reinitialized the variable.